packages/web-app-files/src/composables/actions/files/useFileActionsCreateNewFolder.ts

```diff
@@ -115,12 +115,19 @@ export const useFileActionsCreateNewFolder = ({
     return [
       {
         name: 'create-folder',
-        icon: 'add',
+        icon: 'folder',
         handler,
         label: () => {
-          return $gettext('Create new Folder')
+          return $gettext('New Folder')
         },
-        isEnabled: ({ resources }) => {
+        isEnabled: ({}) => {
+          if (
+            isProjectSpaceResource(space) &&
+            !space.isEditor(store.getters.user) &&
+            !space.isManager(store.getters.user)
+          ) {
+            return false
+          }
```

Comment:
This works for spaces, however, not for shares. You can simply use `space.canUpload()` I believe to cover both cases.

Reply:
oh thanks